packages/vats/src/proposals/upgrade-wallet-factory-proposal.js

```diff
@@ -0,0 +1,58 @@
+// @ts-check
```

Comment:
this is redundant since a couple days ago when this package started linting all JS
```suggestion
```